app/views/hyrax/base/_citations.html.erb

```diff
@@ -1,7 +1,8 @@
 <div class="citations">
-    <% if Hyrax.config.citations? %>
-      <%= link_to t(".citations"), hyrax.citations_work_path(presenter), id: 'citations', class: 'btn btn-default' %>
-    <% end %>
+	<% if Hyrax.config.citations? %>
+		<%= link_to t(".citations"), hyrax.citations_work_path(presenter), id: 'citations', class: 'btn btn-default' %>
+	<% end %>
 
-    Citations: <%= link_to "Download as RIS (Zotero, Endnote, Mendeley)", main_app.polymorphic_path(presenter.solr_document, format: :ris) %>
+	<%= t("work_type.show.citations.label") %>
```

Comment:
What if we put this into some context tag (span?) to allow referencing it with CSS selectors for styling or JS purposes?

Reply:
That sounds like a good idea, but maybe we could skip that for now and do it when we have a strong need?